Automattic/jetpack
# Blocks: use custom SVGs instead of trying to rely on core's icons

Fixes #32704

## Proposed changes:

Using proper SVGs should avoid those accessibility issues.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

1. Go to Posts > Add New
2. Search for Instagram and Facebook blocks, ensure the icons look good:

<img width="592" alt="Screenshot 2023-08-28 at 12 11 24" src="https://github.com/Automattic/jetpack/assets/426388/081647a0-b897-4023-aa78-82ca0ec9c376">
<img width="599" alt="Screenshot 2023-08-28 at 12 11 20" src="https://github.com/Automattic/jetpack/assets/426388/e81a0eba-a412-4bd8-93af-6b5831dfd65c">

Alternatively, you can try this:

1. start a post.
2. open the block inserter sidebar.
3. open F12. (I'm not sure how to do this on my Mac, so I wasn't able to follow those instructions).
4. using accessibility inspector, look for the Instagram block. **You should not see a unicode character before the block name.
